contrib/docker/Dockerfile.downstream

```diff
@@ -1,22 +1,40 @@
-#!/usr/bin/env bash
+ARG TARGETARCH
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.22.5-202407301806.g4c8b32d.el9 as builder
```

Comment:
is that going to be automatically tracked and updated by konflux bot?
currently in midstream we're on go 1.22.7

Reply:
Yes konflux will automatically upgrade it.